src/App.tsx

```diff
@@ -10,33 +12,31 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState('tab-1');
+  const selectedTab: Tab = tabs.find(tab => (
+    tab.id === selectedTabId
+  )) || tabs[0];
+
+  const handleClickTab = (tabId: string) => {
+    if (selectedTabId !== tabId) {
+      setSelectedTabId(tabId);
+    }
+
+    return selectedTabId;
+  };
+
   return (
     <div className="section">
       <h1 className="title">
-        Selected tab is Tab 1
+        {`Selected tab is ${selectedTab.title}`}
       </h1>
 
-      <div data-cy="TabsComponent">
-        <div className="tabs is-boxed">
-          <ul>
-            <li className="is-active" data-cy="Tab">
-              <a href="#tab-1" data-cy="TabLink">Tab 1</a>
-            </li>
-
-            <li data-cy="Tab">
-              <a href="#tab-2" data-cy="TabLink">Tab 2</a>
-            </li>
-
-            <li data-cy="Tab">
-              <a href="#tab-3" data-cy="TabLink">Tab 3</a>
-            </li>
-          </ul>
-        </div>
-
-        <div className="block" data-cy="TabContent">
-          Some text 1
-        </div>
-      </div>
+      <Tabs
+        tabs={tabs}
+        selectedTabId={selectedTabId}
+        onTabSelected={handleClickTab}
+        selectedTab={selectedTab}
```

Comment:
you only need to pass `selectedTabId` and find needed tab inside `Tabs` component, did you run `npm test`?
![image](https://user-images.githubusercontent.com/37161315/229119002-dfe84093-3d48-4777-974c-4867f0086fd4.png)

Reply:
yes, i run, idk why it's doesn't work, i fix this rigth know!